### ISSUE TITLE
Cannot find module '@svgr/webpack'

### ISSUE THREAD

**Issue by dstoyanoff** [teambit/bit#4598]
### Describe the bug
In version 0.0.460 bit throws the following error on build:

```
Cannot find module '@svgr/webpack'
Require stack:
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/react/dist/webpack/webpack.config.base.js
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/react/dist/react.env.js
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/react/dist/react.main.runtime.js
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/bit/dist/load-bit.js
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/bit/dist/app.js
- /Users/danielstoyanoff/.bvm/versions/0.0.460/bit-0.0.460/node_modules/@teambit/bit/bin/bit
```

The issue is not reproducible with 0.0.455

### Specifications
* Bit version: 0.0.460
* Workspace type: harmony
* Node version: 14.17.0
* npm / yarn version: 1.22.10
* Platform: darwin

_for harmony workspace_

* relevant env typescript/react

**Comment by GiladShoham** [comment]
Thanks, we will fix it asap

**Comment by GiladShoham** [comment]
fixed here - https://github.com/teambit/bit/commit/be8d68399177cbc0f58c763f84566a11ec02c1bf
Will be in the next version in about 2 hours

**Comment by KutnerUri** [comment]
confirmed fixed in `v0.0.461`
